### PR TITLE
Adds "antagonist" message when polymorphed into Xenomorph

### PR DESCRIPTION
**Pull request by FloFluoro**
## What Does This PR Do
This PR adds a ~~small~~ message that appears when a player is polymorphed into a Xenomorph:
image

## Why It's Good For The Game
To quote Paradise's advanced rules,
>If you are changed into a xenomorph or terror spiders, your identity is subsumed by the hive - you are expected to follow the orders of Queens, **and are hostile to the crew**. While you may remember or act somewhat as your previous self, **your new loyalty is to the swarm**.

When a Wizard polymorphs someone into a Xenomorph, the xeno is expected by the rules to act like an antagonist, but this isn't mentioned anywhere ingame. Because of this lack of clarity, polymorph xenos often treat their new form as a novelty and avoid antagonistic behavior, or worse, join the crew in hunting down the Wizard. By making it clear IC that polymorph xenos are meant to be hostile to crew, this problem can be avoided.

## Changelog
:cl:
add: Adds antagonist message when polymorphed into a Xenomorph
/:cl:

### CODE REVIEW

**Comment by SpringSkipper** [comment]
I'd make it even bigger and more impossible to miss, just to be sure.

Like, use antag objective text size.

File: code/modules/projectiles/projectile/magic.dm [hunk 1]
```diff
@@ -221,6 +221,7 @@
 				else
 					new_mob = new /mob/living/carbon/alien/humanoid/sentinel(M.loc)
 				new_mob.universal_speak = TRUE
+				to_chat(M, "<b><font color=red>Your consciousness is subsumed by a distant hivemind... you feel murderous hostility towards non-xenomorph life!</font></b>")
```
Comment (Sirryan2002) [inline code/modules/projectiles/projectile/magic.dm:224]:
```suggestion
				to_chat(M, "<span class='warning'>Your consciousness is subsumed by a distant hivemind... you feel murderous hostility towards non-xenomorph life!</span>")
```

Comment (FloFluoro) [reply code/modules/projectiles/projectile/magic.dm]: Done!

File: code/modules/projectiles/projectile/magic.dm [hunk 2]
```diff
@@ -221,6 +221,7 @@
 				else
 					new_mob = new /mob/living/carbon/alien/humanoid/sentinel(M.loc)
 				new_mob.universal_speak = TRUE
+				to_chat(M, "<span class='userdanger'>Your consciousness is subsumed by a distant hivemind... you feel murderous hostility towards non-xenomorph life!<span>")
```
Comment (Sirryan2002) [inline code/modules/projectiles/projectile/magic.dm:224]:
```suggestion
				to_chat(M, "<span class='userdanger'>Your consciousness is subsumed by a distant hivemind... you feel murderous hostility towards non-xenomorph life!</span>")
```

Comment (FloFluoro) [reply code/modules/projectiles/projectile/magic.dm]: Fixed.

**Review by Sirryan2002 — APPROVED** [review]

**Comment by Christasmurf** [comment]
Does this also play the objective alert sound? When you get a new target via cryo or an admin forces your objectives to display, normally there's like a buzzing sort of sound to alert the player, just wondering if this has that as well

**Comment by FloFluoro** [comment]
> Does this also play the objective alert sound?

No, the only thing this PR does is display a message. Antagonist objectives aren't formally assigned to the xenomorph, only a chat notification that you're to act as an antagonist.

**Review by hal9000PR — APPROVED** [review]

**Pull request merged**